src/codecs/md/index.ts

```diff
@@ -627,13 +641,9 @@ function encodeCodeChunk(chunk: stencila.CodeChunk): Extension {
   }
   nodes.push(codeBlock)
 
-  // Separate the `output` with a `ThematicBreak`
   if (outputs && outputs.length) {
-    let index = 0
     for (const output of outputs) {
-      if (index !== 0) nodes.push({ type: 'ThematicBreak' })
```

Comment:
I removed this because it didn't seem to be necessary.

Reply:
The intention of this was to use a `ThematicBreak` as a separator between the individual `outputs` of a `CodeChunk`. Without it, when decoding a `CodeChunk` you can't tell where one output ends and another begins.